internal/util/csiconfig.go

```diff
@@ -164,11 +176,21 @@ func GetClusterID(options map[string]string) (string, error) {
 	return clusterID, nil
 }
 
-func GetNetNamespaceFilePath(pathToConfig, clusterID string) (string, error) {
+func GetRBDNetNamespaceFilePath(pathToConfig, clusterID string) (string, error) {
+	cluster, err := readClusterInfo(pathToConfig, clusterID)
+	if err != nil {
+		return "", err
+	}
+
+	return cluster.RBD.NetNamespaceFilePath, nil
```

Comment:
What if `NetNamespaceFilePath` is empty? Is it handled somewhere?

Reply:
handled here https://github.com/ceph/ceph-csi/blob/devel/internal/rbd/rbd_attach.go#L463-L467